src/rtk/app/store.js

```diff
@@ -1,12 +1,18 @@
 import { configureStore } from "@reduxjs/toolkit";
 import registerSlice from "../features/register/registerSlice";
 import authSlice from "../features/auth/authSlice";
+import blogsSlice from "../features/blogs/blogsSlice";
 
 const store = configureStore({
   reducer: {
     register: registerSlice,
     auth: authSlice,
+    blogs: blogsSlice,
   },
+  middleware: (getDefaultMiddleware) =>
+    getDefaultMiddleware({
+      serializableCheck: false,
```

Comment:
why its needed?

Reply:
The serializableCheck middleware is concerned with ensuring that the Redux actions dispatched contain serializable payloads. But here we are fetching data from localstorage that is already serializable. So we don't need to use it.